faasmcli/faasmcli/util/call.py

```diff
@@ -21,11 +23,55 @@ def _do_invoke(user, func, host, port, func_type, input=None):
     do_post(url, input, json=True)
 
 
+def _async_invoke(url, msg, headers=None, poll=False, host=None, port=None):
```

Comment:
This is fixing a PEP warning where the complexity of the `invoke_impl` function was too high